### PR TITLE
Added 3 Design Resources

### PR DESCRIPTION
**Pull request by 01-DC**
- Added remove.bg, ls.graphics and Unsplash. Very useful tools for designers and video editors to get things done quickly.
- Added description for unDraw

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -588,6 +588,9 @@ To add a resource or contribute, please check out our contribution guide [here](
 - [3D-Icons](https://free3dicon.com/): Free 3D - Icons to download
 - [533 Hand-Picked Tools & Resources for Web Designers & Developers](https://toolkit.addy.codes/)
 - [Illlustrations](https://illlustrations.co/): Awesome 120+ free illustrations
+- [remove.bg](https://www.remove.bg/): Remove background from images and videos quickly
```
Comment (shahednasser) [inline README.md:591]: There are tools that are provide better features for free, so please remove this one

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your work @01-DC I just have a comment below

**Comment by 01-DC** [comment]
Thank you for the suggestion @shahednasser I have included the changes. Please review it. I am a little new to open-source contributions. Sorry for any mistakes.

**Comment by shahednasser** [comment]
Thank you @01-DC don't worry, you didn't make any mistakes!

**Pull request merged**